src/transformer/mockDefiner/mockDefiner.ts

```diff
@@ -152,24 +146,22 @@ export class MockDefiner {
     return this._getCallGetFactory(key);
   }
 
-  public getDeclarationKeyMap(declaration: ts.Declaration): string | undefined {
+  public getDeclarationKeyMap(declaration: ts.Declaration): string {
+    let key: string | undefined;
+
     if (!this._declarationCache.has(declaration)) {
-      const key: string = this._factoryUniqueName.createForDeclaration(declaration as PossibleDeclaration);
+      key = this._factoryUniqueName.createForDeclaration(declaration as PossibleDeclaration);
 
       this._declarationCache.set(declaration, key);
     }
 
-    return this._declarationCache.get(declaration);
+    // NOTE: TypeScript does not support inference through has/get, but we know
```

Comment:
Instead of creating a let variable, could you just add the casting to the return and keep the code as it was ?

```
return this._declarationCache.get(declaration) as string;

```